src/control/SConscript

```diff
@@ -103,18 +103,26 @@ def scons():
     gospdkpath = join(gosrc, "vendor", "github.com", "daos-stack", "go-spdk",
                       "spdk")
 
-    # CGO prerequisite libs for nvme
+    # CGO prerequisite libs for go-spdk bindings
     denv.AppendUnique(CPPPATH=[join(gospdkpath, "include")])
-    nvmecontrol = denv.StaticLibrary(
-        "nvme_control",
-        join(gospdkpath, "src", "nvme_control.c"),
-        LIBS=['spdk'])
-    nc_installed = denv.Install(join(denv.subst("$PREFIX"), "lib"), nvmecontrol)
+    # hack to avoid building this library with cov compiler for the moment
+    if denv.get('COMPILER').lower() == "covc":
```

Comment:
What if you just do

cc = denv.get(“COMPILER”)
if cc == “covc”:
    cc =“gcc”

Then you can do

StaticLibrary(...., CC=cc, ...)